object_manager.go

```diff
@@ -26,20 +26,31 @@ type IBObjectManager interface {
 	GetEADefinition(name string) (*EADefinition, error)
 	CreateEADefinition(eadef EADefinition) (*EADefinition, error)
 	UpdateNetworkViewEA(ref string, addEA EA, removeEA EA) error
-	CreateHostRecord(enabledns bool, recordName string, netview string, dnsview string, cidr string, ipAddr string, macAddress string, vmID string, vmName string) (*HostRecord, error)
+}
+
+type IBOjectManager1 interface{
```

Comment:
why this change has been introduced ?

Reply:
the interface was becoming big hence added these changes

Reply:
please take it as separate PR. we can create a interface for each object type. And group it in the objectManager interface

Reply:
Sure then for now will add everything under one object?

Reply:
@chinmayb  should i add the functions back under the first inteface itself and then raise a separate PR to differentiate them?

Reply:
yes, you can do that